apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/main/java/gov/cms/bfd/pipeline/rda/grpc/PreAdjudicatedClaim.java

```diff
@@ -0,0 +1,4 @@
+package gov.cms.bfd.pipeline.rda.grpc;
+
+/** Java bean representing preadjudicated claims received from RDA API. */
+public class PreAdjudicatedClaim {}
```

Comment:
I'm a little iffy on this choice, similar to the conversation you & Eric were having elsewhere. I think it's fine for now, but I'll be curious to see how it plays out once you actually start catching data.

For one thing, I prolly wouldn't assume this early that you'll be _able_ to have a single base class for all the various data types that you might end up having to catch & process.